test/e2e/scenarios/multiple_clusters.go

```diff
@@ -0,0 +1,27 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("[Dataplane] Namespace selector based Network Policies for ingress between clusters, selecting multiple remote clusters", func() {
+	PIt("Should allow communication between pods in selected namespace across multiple clusters", func() {
+		By("creating listener pod 1 with label 1 in cluster 1 in namespace 1")
+		By("creating connecting pod 2 with label 2 in cluster 2 in namespace 2")
+		By("creating connecting pod 3 with label 2 in cluster 3 in namespace 2")
+		By("creating network policy in cluster 1 allowing pods in namespace 2 with label 2 on more than 2 clusters to communicate with pods in namespace 1")
```

Comment:
This PDescribe talks about "Namespace selector" , not namespace + label ?

Reply:
Removed label reference from policy creation